package-lock.json

```diff
@@ -1205,6 +1205,7 @@
       "integrity": "sha512-UomYWcCpM7OZUt1BDlY3guO6mnA4VXzMkNjFbVtWibKQkk4LhcIUXb6SxWSw/gujIrlOZywldjyj8bL6V374IQ==",
       "dev": true,
       "optional": true,
+      "peer": true,
```

Comment:
this looks unrelated to the update of mongodb. perhaps it's now using a newer version of npm? anyhow if we want this added, probably better to have as a separate PR.

Reply:
Agreed that this doesn't make sense. 

> perhaps it's now using a newer version of npm?

Seems like a good guess. I don't have any other ideas.